tests/unit/validation/test_validation_functions.py

```diff
@@ -431,6 +434,177 @@ def test_validate_values__infinite_sigmas(sensor_type, parameter):
         assert not isinstance(error, InfinityError)
 
 
+@pytest.mark.parametrize(
+    ("sensor_type", "parameter", "values", "error_types"),
+    [
+        ("sym_power_sensor", ["p_sigma", "q_sigma"], [np.nan, np.nan], [InvalidIdError, NotUniqueError]),
+        (
+            "sym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [0.1, np.nan],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "sym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [np.nan, 0.1],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        ("sym_power_sensor", ["p_sigma", "q_sigma"], [0.1, 0.1], [InvalidIdError, NotUniqueError]),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[np.nan, np.nan, np.nan], [np.nan, np.nan, np.nan]],
+            [InvalidIdError, NotUniqueError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[0.1, np.nan, 0.1], [np.nan, 0.1, np.nan]],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[0.1, np.nan, np.nan], [np.nan, np.nan, np.nan]],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[np.nan, np.nan, np.nan], [0.1, np.nan, np.nan]],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[0.1, 0.1, 0.1], [np.nan, np.nan, np.nan]],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[np.nan, np.nan, np.nan], [0.1, 0.1, 0.1]],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[np.nan, np.nan, np.nan], [0.1, np.nan, np.nan]],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[0.1, np.nan, np.nan], [0.1, np.nan, np.nan]],
+            [InvalidIdError, NotUniqueError, MultiFieldValidationError],
+        ),
+        (
+            "asym_power_sensor",
+            ["p_sigma", "q_sigma"],
+            [[0.1, 0.1, 0.1], [0.1, 0.1, 0.1]],
+            [InvalidIdError, NotUniqueError],
+        ),
+    ],
+)
+def test_validate_values__bad_p_q_sigma(sensor_type, parameter, values, error_types):
```

Comment:
i don't think you need the `parameter` field. instead you can provide `p_sigma` and `q_sigma` values as separate input fields. might also make things a little bit more clear

```suggestion
def test_validate_values__bad_p_q_sigma(sensor_type, p_sigma, q_sigma, error_types):
```